### PR TITLE
[Bogdan Pana] unordered_associative_task_1

### PR DESCRIPTION
**Pull request by BogdanPana26**

### CODE REVIEW

File: unordered_associative_task_1/student/include/impl.h [hunk 1]
```diff
@@ -32,4 +33,19 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist)
+{
+    std::unordered_multiset<std::string> first_playlistset;
+    std::unordered_multiset<std::string> second_playlistset;
+
+    for( auto &song : first_playlist)
```
Comment (smay1613) [inline unordered_associative_task_1/student/include/impl.h]: you may directly use constructor with two iterators

File: unordered_associative_task_1/student/include/impl.h [hunk 2]
```diff
@@ -32,4 +33,19 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist)
+{
+    std::unordered_multiset<std::string> first_playlistset;
```
Comment (smay1613) [inline unordered_associative_task_1/student/include/impl.h]: please work with Song abstraction

File: unordered_associative_task_1/student/include/impl.h [hunk 3]
```diff
@@ -32,4 +49,10 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist)
+{
+    std::unordered_multiset<Song> first_playlistset(first_playlist.begin(),first_playlist.end());
```
Comment (smay1613) [inline unordered_associative_task_1/student/include/impl.h:54]: also you could check the size before the copying

Comment (BogdanPana26) [reply unordered_associative_task_1/student/include/impl.h:54]: okk

**Pull request closed without merging**